[icons] feat: new icons for low & high voltage power poles

RexTheInnkeeper opened the pull request:
#### Fixes #0000

#### Checklist

- [ ] Includes tests
- [ ] Update documentation

<!-- DO NOT enable CircleCI for your fork. Our build will run when you open this PR. -->

#### Changes proposed in this pull request:

<!-- . -->
added low and high voltage power pole icons

#### Reviewers should focus on:

<!-- . -->
make sure everything renders ok and correct files were changed

#### Screenshot

<!-- Include an image of the most relevant user-facing change, if any. -->
![Poles 20px](https://user-images.githubusercontent.com/107489230/196798469-eca28cdb-e40d-4d6b-816b-a3e8a803e2b8.png)
![Poles](https://user-images.githubusercontent.com/107489230/196798471-ca63a99f-e4b2-4569-97dd-800db5bb371d.png)

RexTheInnkeep committed:
feat: new icon - power poles

palantirtech commented:
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @RexTheInnkeeper! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1,5 +1,5 @@
 [
-    {
+   {
```
```suggestion
    {
```

adidahiya reviewed (changes requested):
these icons don't show up correctly in the [docs preview build](https://output.circle-artifacts.com/output/job/ad292ee8-d0f6-41bc-a746-270353974a59/artifacts/0/packages/docs-app/dist/index.html#icons):

<img width="277" alt="image" src="https://user-images.githubusercontent.com/723999/197040025-b60e1242-cb26-47c8-b2fe-a335db71f24e.png">

<img width="270" alt="image" src="https://user-images.githubusercontent.com/723999/197040089-c0ac69fd-8fa6-4761-8b74-b962d49dd1c2.png">

RexTheInnkeeper committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

RexTheInnkeep committed:
SVG poor format fix. Category update.

Updated SVG format to be flattened figma svg exports to deal with
incorrect display in preview. Updated category of new icons in
anticipation of additional utility icons entering blueprint.

RexTheInnkeep committed:
Update SVG Formats again

RexTheInnkeeper committed:
Merge remote-tracking branch 'upstream/develop' into rexUtilityIcons

RexTheInnkeeper committed:
Typo Fix

RexTheInnkeeper committed:
Merge branch 'palantir:develop' into rexUtilityIcons

RexTheInnkeeper committed:
Merge branch 'rexUtilityIcons' of https://github.com/RexTheInnkeeper/blueprint into rexUtilityIcons

adidahiya commented:
@RexTheInnkeeper there are still some issues with the 20px high voltage icon:

<img width="257" alt="image" src="https://user-images.githubusercontent.com/723999/197064166-3c9fb95d-1f28-46a9-a481-14099bc8539c.png">

adidahiya commented on resources/icons/20px/high-voltage-pole.svg:
```diff
@@ -0,0 +1,10 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_2240_7)">
```
I think you need to remove this `<g>`

adidahiya commented on resources/icons/20px/high-voltage-pole.svg:
```diff
@@ -0,0 +1,10 @@
+<g clip-path="url(#clip0_2240_7)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M7 0C6.44772 0 6 0.447716 6 1V4H4V2C4.55228 2 5 1.55228 5 1C5 0.447715 4.55228 0 4 0H1C0.447715 0 0 0.447715 0 1C0 1.55228 0.447715 2 1 2H2V4H1C0.447715 4 0 4.44772 0 5C0 5.55228 0.447716 6 1 6H2V8H1C0.447715 8 0 8.44771 0 9C0 9.55228 0.447715 10 1 10H4C4.55228 10 5 9.55228 5 9C5 8.44771 4.55228 8 4 8V6H6V19C6 19.5523 6.44772 20 7 20C7.41998 20 7.7 ...
+</g>
+<defs>
```
also remove these unnecessary `<defs>`

adidahiya committed:
Update high-voltage-pole.svg

adidahiya reviewed (approved):
looks good now:

<img width="351" alt="image" src="https://user-images.githubusercontent.com/723999/197205266-f6a4e27e-60dd-40c9-bed5-d22be476def6.png">

I'm not sure if we'll keep a distinct category for "Utilities" icons like this, but it's fine for now. I'll probably reorganize some icons into a category for "map markers" or something like that.
